projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -29,6 +29,25 @@
 .jp-carousel-wrap.swiper-container {
 	height: auto;
 	width: 100vw;
+	opacity: 0;
+}
+
+.jp-carousel-fade-in-image {
+	animation: fadeInOpacity ease-in 0.5s 1 normal forwards;
```

Comment:
I would be inclined to go with 0.2s - 200ms is quite a good timing for the eyes not to see a harsh switch, but not to notice it as a slow fade, maybe just my opinion, but I think the likes of material design recommend 150-250ms for transitions.

Reply:
How does `0.25s` grab you?